### DISCUSSION TITLE
How to save the predicted labels?

### DISCUSSION THREAD

**Discussion by andressrodrl** [obss/sahi#909] [Q&A]
Is there any predict function argument to save the predicted labels .txt?

**Comment by jokober** [comment]
You can check this issue: https://github.com/obss/sahi/discussions/662

**Reply by andressrodrl** [reply]
I dont get how can i get the yolo .txt annotations with the pickle file, any source on that? But i kinda managed to get it by creating a json with coco style annotations and then using the predict function argument 'dataset_json_path', then I get the annotations in coco format and can convert it to yolo/whatever to calculate different metrics
